covid19dp_submission/download_analyses.py

```diff
@@ -104,11 +96,19 @@ def get_analyses_to_process(project, num_analyses, total_analyses, processed_ana
 
 
 @retry(logger=logger, tries=4, delay=120, backoff=1.2, jitter=(1, 3))
-def get_analyses_from_ena(project, offset, limit):
+def get_analyses_from_ena(project, offset=0, limit=0):
+    """
+    Retrieve analysis of a specific project from ENA API. If offset and limit are not set, it retrieves all the
+    analysis of the project.
+    """
     analyses_url = (
-        f"https://www.ebi.ac.uk/ena/portal/api/filereport?result=analysis&accession={project}&offset={offset}"
-        f"&limit={limit}&format=json&fields=run_ref,analysis_accession,submitted_ftp,submitted_aspera,tax_id"
+        f"https://www.ebi.ac.uk/ena/portal/api/filereport?result=analysis&accession={project}"
+        f"&format=json&fields=run_ref,analysis_accession,submitted_ftp,submitted_aspera,tax_id"
     )
+    if offset:
+        analyses_url += f'&offset={offset}'
+    if limit:
```

Comment:
Do we get all the records if we don't put **limit** param in the URL ?

Reply:
I think you're right. It's not retrieving the full set only a default of 100K. And that's why the test are passing in a reasonable amount of time.
I need to specifically set the `limit=0` to get the full set. And it is slooow !

Reply:
I checked locally and the test now take >25 mins